presentation-compiler/test/dotty/tools/pc/tests/completion/CompletionSuite.scala

```diff
@@ -713,8 +719,8 @@ class CompletionSuite extends BaseCompletionSuite:
           |}
           |""".stripMargin,
       """|Number: Regex
-         |Nil:  scala.collection.immutable
```

Comment:
Why is there sometimes a ":" and sometimes not?

Reply:
It depends on whether we use a normal standard library or a bootstrapped one.
A default scala standard library, returns different canditate for completion.
In case of standard library, it returns an `ExprType` of Nil. Scala stdlib-bootstrapped on the other hands, returns a `TermRef` for value Nil, thus has different label is created.

The deletion of ":" in those tests is caused, because I moved back from stdlib-bootstrapped to standard one as I no longer need it for tests.

Reply:
> In case of standard library, it returns an ExprType of Nil. Scala stdlib-bootstrapped on the other hands, returns a TermRef for value Nil, thus has different label is created.

Does that impact what is shown to the user or is this an implementation detail? It seems like the tests would be more stable and easier to read if the ':' was always present.

Reply:
Sounds like an issue with the printer then?

Reply:
To be honest, I'm not sure if this is an issue with the printer or rather the fact that Scala 2 library unpickled symbols are different from symbols that come from Scala 2 library compiled with Scala 3 aka stdlib-bootstrapped.

Reply:
I don't know why the symbol types are different (maybe because objects are represented slightly differently in Scala 2 and 3) but in general I don't think that whether the type of something is an `ExprType` or a `TermRef` should affect what the pretty-printer does.

Reply:
I see, so this is a bug in printer indeed. As it was present in metals before, I propose that we create an issue and handle this in the next PR as this one is already enormous.